rotkehlchen/chain/ethereum/decoding/decoder.py

```diff
@@ -597,11 +597,23 @@ def _maybe_enrich_transfers(  # pylint: disable=no-self-use
         if tx_log.topics[0] == GNOSIS_CHAIN_BRIDGE_RECEIVE and tx_log.address == '0x88ad09518695c6c3712AC10a214bE5109a655671':  # noqa: E501
             for event in decoded_events:
                 if event.event_type == HistoryEventType.RECEIVE:
+                    try:
+                        crypto_asset = event.asset.resolve_to_crypto_asset()
+                    except (UnknownAsset, WrongAssetType) as e:
+                        log.error(
```

Comment:
you don't need both a log.error() and a msg_aggregator.add_error as the latter will also by default make a log entry.

Additionally what you can do here is reuse the notify_user. Just use any decoder from `self.decoders` and get the `notify_user` from there.

Something like `next(iter(self.decoder.values())).notify_user` ?